04/app.js

```diff
@@ -2,8 +2,27 @@ const user = {
     firstName: 'Adam',
     lastName: 'Nowak',
     born: {
-        day: '14',
-        month: '04',
-        year: '1985'
+        day: '21',
+        month: '03',
+        year: '1984'
     }
-}
\ No newline at end of file
+}
+
+user.getBirthdayDay = function() {
+
+    const today = new Date().toLocaleDateString();
+    const date = this['born'];
+    const baseDate =  date['day'] + "." + date['month'] + "." + date['year'];
```

Comment:
To oznacza, że nie zawsze tak będzie wyglądać ten ciąg znaków jak zaproponowałeś.
Dlatego to nie na każdym komputerze zadziała - jeszcze gorsze jest to, że ten błąd trudno zlokalizować bo pewnie "u Ciebie" to będzie działać :D